docs/authentication.md

```diff
@@ -0,0 +1,141 @@
+Authentication
+==============
+
+Charmed Kubeflow uses [Dex][dex] for authentication. Dex allows for pluggable authentication
+against many different identity providers, while presenting a unified
+[OpenId Connect (OIDC)][oidc] interface.
+
+[dex]: https://github.com/dexidp/dex
+[oidc]: https://en.wikipedia.org/wiki/OpenID_Connect
+
+
+Auth Flow
+---------
+
+The authentication flow for an unauthenticated request consists of these steps:
+
+ - User makes an unauthenticated request
+   - Redirected to /dex
+ - User logs in via Dex
+ - Redirected to gatekeeper callback endpoint
+   - Sets a cookie with auth token for future requests
+ - Redirected back to original page, with authorization token
+
+### Unauthenticated Request
+
+The flow of an unauthenticated request is shown here:
+
+<img
+    src="https://raw.githubusercontent.com/juju-solutions/bundle-kubeflow/dex-auth-flow/docs/img/dex-unauthenticated.svg"
```

Comment:
Works fine for me on GH.

Reply:
@knkski pointed out that it's actually the logos embedded in the image that don't work, rather than the base image, and they do not, in fact, show up for me on GH.